medicines/pars-upload/src/products_form.js

```diff
@@ -92,7 +92,7 @@ export const Products = ({
           <FormGroup key={i}>
             <Field
               name="active_substance"
-              label="Active substance"
+              label="Active substance(s)"
```

Comment:
@roughprada are we also ditching the buttons to add extra Active Substances? also, note that the tests are failing because they can't find this label now

Reply:
I think I've replaced only the test.js one. right? we are not ditching it, do you see this incorrect? I was thinking that I might add more than 1 active substance and when we add an extra one, we don't show label, only the field underneath

Reply:
If this repeats as you add more, it might look really weird